jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -633,23 +706,36 @@ impl Epoch {
         let lower_bound = context
             .db
             .get_block(&epoch_data.first_block)
-            .map(|block| block.expect("The block to be indexed").chain_length)
+            .map(|block| u32::from(block.expect("The block to be indexed").chain_length))
```

Comment:
Why are block information structs convertible to integer indices? A `From` impl does not make sense here and might cause confusion.

Reply:
what structs? `ChainLength` is a newtype wrapping a `u32`

Reply:
Right, I did not read this carefully enough, there's a field access at the end of it.
The result of `get_block` should be meant to be lightweight enough for this to be good going forward, otherwise there should be data store methods to query the header information specifically.

Reply:
I'm not really storing the header (`get_block` gives you an `ExplorerBlock`, which is a version with only the things the API is exposing), if blocks get too big it may be better to do what you say, but I'm not sure if I should address that now, as most blocks won't have that many transactions. (and they are stored in memory)